dashboard/app/controllers/survey_results_controller.rb

```diff
@@ -17,7 +17,8 @@ def create
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def survey_result_params
-    params.require(:survey).permit(SurveyResult::RESULT_ATTRS.map(&:to_sym))
+    params.require(:survey).
+      permit(SurveyResult::RESULT_ATTRS.map(&:to_sym) + [:kind])
```

Comment:
My thought would be to add `nps_value` and `nps_comment`, though I'm happy to remove the "nps" prefix if you think I should (seeing as `kind` will be `NetPromoterScore`) so as to make them more reusable (a good thing or bad thing?).